builtin/mainmenu/content/dlg_package.lua

```diff
@@ -172,43 +159,33 @@ local function get_formspec(data)
 
 		if current_tab == 1 then
 			-- Screenshots and description
-			local hypertext = info.long_description.head ..
-					"<big><b>" .. core.hypertext_escape(info.short_description) .. "</b></big>\n\n" ..
+			local hypertext = "<big><b>" .. core.hypertext_escape(info.short_description) .. "</b></big>\n"
+			local winfo = core.get_window_info()
+			local fs_to_px = winfo.size.x / winfo.max_formspec_size.x
+			for i, ss in ipairs(info.screenshots) do
+				local path = get_screenshot(data.package, ss.url, 2)
+				hypertext = hypertext .. "<action name=ss_" .. i .. "><img name=" ..
+						core.hypertext_escape(path) .. " width=" .. (3 * fs_to_px) ..
+						" height=" .. (2 * fs_to_px) .. "></action>"
+				if i ~= #info.screenshots then
+					hypertext = hypertext .. "<img name=blank.png width=" .. (0.25 * fs_to_px) ..
+							" height=" .. (2.25 * fs_to_px).. ">"
+				end
+			end
+			hypertext = hypertext .. "\n" .. info.long_description.head ..
 					info.long_description.body
 
 			hypertext = hypertext:gsub("<img name=blank.png ",
 					"<img name=" .. core.hypertext_escape(defaulttexturedir) .. "blank.png ")
 
-			local screenshots_w = #info.screenshots*3.25 + 0.375 - 0.25
-			local scroll_factor = 0.1
-			local needs_scrollbar = screenshots_w > size.x
-			local hypertext_y = (#info.screenshots > 0 and 2.25 or 0) + (needs_scrollbar and 0.5 or 0)
-
 			table.insert_all(formspec, {
-				"hypertext[0.375,", hypertext_y, ";",
-					size.x - 2*0.375, ",",
-					tab_body_height - hypertext_y - 0.375,
+				"hypertext[0.375,0;",
+					size.x - 3*0.375, ",",
```

Comment:
I changed `2*0.375` -> `3*0.375` to fix this:

>  The package description scrollbar doesn't work on my phone because it's in the "no touch zone" near the edge of the screen.

It needs to be applied to the `elseif current_tab == 2 then` branch too, I missed that.